File: Vb.net/StoryDownload/StoryDownload/Form1.vb
Imports System.IO, System.Net, System.Text, System.Text.RegularExpressions
Public Class Form1

    Private Sub Form1_Load(sender As Object, e As EventArgs) Handles MyBase.Load
        MessageBox.Show("This Project Coded By > MrVirus" + vbCrLf + "Instagram : BQBB" + vbCrLf + "Telegram : camera")
        CheckForIllegalCrossThreadCalls = False
    End Sub
    Function getid(user As String)
        Dim web As New WebClient
        Dim result As String = web.DownloadString("https://www.instagram.com/" + user)
        Dim idu As String = Regex.Match(result, """profilePage_(.*?)""").Groups(1).Value
        Return idu
    End Function
    Function downloadall(idu As String)
        Dim path As String = System.Environment.CurrentDirectory

        Dim web As New WebClient
        Dim download As New WebClient
        Dim result As String = web.DownloadString("http://mrvirus.cf/StoryDownload.php?id=" + idu)
        Dim ma As MatchCollection = Regex.Matches(result, "{URL:(.*?)}")
        Dim ib As Integer = 0
        For Each url As Match In ma
            Dim ul As String = url.Groups(1).Value
            If ul.Contains("mp4") Then
                Dim ul2 As Uri = New Uri(ul)

                download.DownloadFile(ul2, path + "\" + TextBox1.Text.ToString() + ib.ToString + ".mp4")
                ib += 1
            Else
                Dim ul2 As Uri = New Uri(ul)
                download.DownloadFile(ul2, path + "\" + TextBox1.Text.ToString() + ib.ToString + ".jpg")
                ib += 1
            End If

        Next
        MsgBox("<3")
        Return True
    End Function

    Private Sub Button1_Click(sender As Object, e As EventArgs) Handles Button1.Click
        Dim ids As String = getid(TextBox1.Text)
        downloadall(ids)

    End Sub
End Class

File: C#/StoryDownload - C/StoryDownload - C/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;
using System.Text.RegularExpressions;
using System.Net;
using System.IO;

namespace StoryDownload___C
{
    public partial class Form1 : Form
    {
        

       

        public Form1()
        {
            InitializeComponent();
        }

        private void Form1_Load(object sender, EventArgs e)
        {
            MessageBox.Show("This Project Coded By > MrVirus\nInstagram : BQBB\nTelegram : camera");
        }
        public static string getid(string user)
        {

            WebClient web = new WebClient();
          
            var result = web.DownloadString("https://www.instagram.com/" + user);
            string ids=Regex.Match(result, ":\"profilePage_(.*?)\",").Groups[1].Value;
            return ids;
        }
        public void  downloader(string id)
        {
            string Path = System.Environment.CurrentDirectory;
            string url = "http://mrvirus.cf/StoryDownload.php?id="+id;
            WebClient web = new WebClient();
            WebClient download = new WebClient();
            string result = web.DownloadString(url);
            MatchCollection re = Regex.Matches(result, "{URL:(.*?)}");
            int ib = 0;
            foreach (Match x in re)
            {
                if (x.Groups[1].Value.Contains("mp4"))
                {
                    download.DownloadFile(x.Groups[1].Value, Path+"\\vid" + textBox1.Text + Convert.ToString(ib) + ".mp4");
                    ib += 1;

                }
                else
                {
                    download.DownloadFile(x.Groups[1].Value, Path + "\\Photo" + textBox1.Text + Convert.ToString(ib) + ".jpg");
                     ib += 1;
                }
            }
        }

        private void button1_Click(object sender, EventArgs e)
        {
            string id = getid(textBox1.Text);
            downloader(id);


        }
    }
}
